test/blackbox-tests/test-cases/pkg/toolchain-installation.t

```diff
@@ -0,0 +1,95 @@
+Test the installation of toolchains package by building and installing
+a mock compiler package using dune's toolchain mechanism.
+
+  $ . ./helpers.sh
+  $ make_lockdir
+
+  $ mkdir fake-compiler
+
+A configure script for the fake compiler that writes the prefix path
+to a file so the fake compiler can be installed to the specified
+location, testing that dune respects the "prefix" variable for
+toolchain packages.
+  $ cat > fake-compiler/configure << EOF
```

Comment:
Nitpick but if you [quote EOF](https://linuxize.com/post/bash-heredoc/) you don't have to escape `$` which will make it a bit easier to read and extend since one doesn't have to remember to add the escapes.